extensions-core/druid-bloom-filter/src/main/java/org/apache/druid/query/aggregation/bloom/types/DoubleBloomFilterAggregatorColumnSelectorStrategy.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.aggregation.bloom.types;
+
+import org.apache.druid.common.config.NullHandling;
+import org.apache.druid.segment.DoubleColumnSelector;
+import org.apache.hive.common.util.BloomKFilter;
+
+public class DoubleBloomFilterAggregatorColumnSelectorStrategy
+    implements BloomFilterAggregatorColumnSelectorStrategy<DoubleColumnSelector>
```

Comment:
`DoubleColumnSelector` must appear only in `implements` clauses and nowhere else. See it's Javadoc. Same for float and long.